src/Extensions/NumberExtensions.ts

```diff
@@ -0,0 +1,16 @@
+declare global {
+    interface Number {
+        toPercentString(): string;
+    }    
+}
+
+/**
+ * Meant for numbers between 0 and 1 (but doesn't have to be)
+ * Multiplies by 100 and goes to 1 decimal place
+ */
+Number.prototype.toPercentString = function (): string {
+    // Number vs number discrepancy
+    return (this as number * 100).toFixed(1);
```

Comment:
Should this have a `%` after it if it's a string?

Reply:
yeah, I'm down with that